keras_cv/models/object_detection/yolo_v8/yolo_v8_backbone_presets.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""YOLOv8 Backbone presets."""
+
+backbone_presets_no_weights = {
+    "yolov8_xs_backbone": {
+        "metadata": {
+            "description": "An extra small YOLOV8 backbone",
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "channels": [32, 64, 128, 256],
+            "depths": [1, 2, 2, 1],
+            "activation": "swish",
+        },
+    },
+    "yolov8_s_backbone": {
+        "metadata": {
+            "description": "A small YOLOV8 backbone",
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "channels": [64, 128, 256, 512],
+            "depths": [1, 2, 2, 1],
+            "activation": "swish",
+        },
+    },
+    "yolov8_m_backbone": {
+        "metadata": {
+            "description": "A medium YOLOV8 backbone",
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "channels": [96, 192, 384, 576],
+            "depths": [2, 4, 4, 2],
+            "activation": "swish",
+        },
+    },
+    "yolov8_l_backbone": {
+        "metadata": {
+            "description": "A large YOLOV8 backbone",
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "channels": [128, 256, 512, 512],
+            "depths": [3, 6, 6, 3],
+            "activation": "swish",
+        },
+    },
+    "yolov8_xl_backbone": {
+        "metadata": {
+            "description": "An extra large YOLOV8 backbone",
+        },
+        "class_name": "keras_cv.models>YOLOV8Backbone",
+        "config": {
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "channels": [160, 320, 640, 640],
+            "depths": [3, 6, 6, 3],
+            "activation": "swish",
+        },
+    },
+}
+
+backbone_presets_with_weights = {
+    "yolov8_xs_backbone_coco": {
```

Comment:
Should we rename to "yolov8_xs_csp_darknet_coco" or even just remove the "backbone" piece of this string identifier?

Reply:
Ah wait; this is for the backbone itself.